src/ios/TLRKFeedback.m

```diff
@@ -34,6 +39,36 @@ -(void)initialize: (CDVInvokedUrlCommand *)command
         uid = objc_msgSend(objc_msgSend(UIDevice.currentDevice, identifierForVendorSEL), @selector(UUIDString));
     }
 
+	// iOS localization specific options
+    // TODO: refactor the code for DRY issues
+    // using UTF8 strings to allow displaying Chinese, Cyrillic, etc. chars 
+    // using NSNull comparison for JSON values set to null
+	if(command.arguments.count == 3 && command.arguments[2]) {
+		NSDictionary *feedbackOptions = command.arguments[2];
+
+		NSDictionary *iOSLocalizationOptions = [feedbackOptions valueForKey:iOSFeedbackOptionsKeyName];
```

Comment:
I'd prefer this syntax to make sure that we if the value is not there, we get nil (even though this was already checked). For me the current state better exemplifies the checks we are making there. Otherwise we assume that the members do not contain nil values.